psi4/src/psi4/libmints/matrix.cc

```diff
@@ -237,6 +237,29 @@ Matrix::Matrix(dpdfile2 *inFile)
     global_dpd_->file2_mat_close(inFile);
 }
 
+Matrix::Matrix(dpdbuf4 *inBuf)
+    : name_("None"), rowspi_(inBuf->params->nirreps), colspi_(inBuf->params->nirreps)
+{
+    matrix_ = NULL;
+    symmetry_ = 0; //inBuf->my_irrep;
```

Comment:
Is there a reason you set `symmetry_` to 0? The commented `inbuf->my_irrep` should be the correct value.